packages/wonder-stuff-render-server/types/superagent-cache-plugin.d.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 import {SuperAgentRequest, Response} from "superagent";
```

Comment:
The eslint rule thinks `SuperAgentRequest` isn't being used because it doesn't understand interface merging. 😞